public/app/features/live/LiveConnectionWarning.tsx

```diff
@@ -45,8 +45,8 @@ export class LiveConnectionWarning extends PureComponent<Props, State> {
   render() {
     const { show } = this.state;
     if (show) {
-      if (!contextSrv.isSignedIn) {
-        return null; // do not show the warning for anonomous users (and /login page etc)
+      if (!contextSrv.isSignedIn || !config.liveEnabled) {
```

Comment:
This only shows up in dev mode, so it is not urgent...

BUT we may also want to check the disconnect reason to know if we should really show an error or not